test/acceptance/pages.py

```diff
@@ -512,8 +511,6 @@ def available_checked_out_numbers(self):
         Gets "N available and M checked out" information from staff grading sections.
         Returns tuple of (N, M)
         """
-        if not 'GRADE AVAILABLE RESPONSES' in self.selected_button_names:
```

Comment:
Note to reviewers: I wanted to be able to get the counts before we expand the grading form. Therefore the test points explicitly call self.click_staff_toolbar_button("staff-grading") when necessary.